### PR TITLE
Gal-19 classificar titulos virtuais catalogo pergamum

### PR DESCRIPTION
**Pull request by matheussouza9**

### CODE REVIEW

File: aplicacoes/gal_npi/src/main/java/br/ufc/npi/gal/model/Titulo.java [hunk 1]
```diff
@@ -76,6 +76,9 @@ public Titulo(String nome, String isbn, String tipo) {
 	
 	@Column(name = "publicador")
 	private String publicador;
+	
+	@Column(name = "cadastrado_biblioteca")
+	private Boolean cadastradoBiblioteca;
```
Comment (brunomateus) [inline aplicacoes/gal_npi/src/main/java/br/ufc/npi/gal/model/Titulo.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/NPI-UFC/gal/file/2400694405/issues/source?bid=3232095&fileBranchId=3232273#l81)

File: aplicacoes/gal_npi/src/main/webapp/resources/css/gal.css [hunk 2]
```diff
@@ -151,15 +151,19 @@ div#container {
...
 	padding-bottom: 9px !important;
 	background-color: #d3d6ff !important;
 }
-.disciplina-panel-body{
+.componente-bibliografia-panel-body{
 	background-color: #eaebff !important;
+}
+
+.bibliografia-virtual-nao-cadastrada{
+	background-color: #ffcc00 !important;
```
Comment (brunomateus) [inline aplicacoes/gal_npi/src/main/webapp/resources/css/gal.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/NPI-UFC/gal/file/2400693959/issues/source?bid=3232095&fileBranchId=3232273#l168)

Comment (npiuser) [inline aplicacoes/gal_npi/src/main/webapp/resources/css/gal.css]: Codacy Issue found: [Use of !important](https://www.codacy.com/app/NPI-UFC/gal/file/2400813161/issues/source?bid=3232095&fileBranchId=3232273#l168)

File: aplicacoes/gal_npi/src/main/webapp/WEB-INF/views/titulo/adicionar.jsp [hunk 3]
```diff
@@ -108,14 +109,21 @@
 			<div class="form-group">
 			    <label for="tipo" class="col-sm-1 control-label">Tipo</label>
 			    <div class="col-sm-10">
-			    	<form:select path="tipo" class="form-control">
-			    		<form:option value="Físico">Físico</form:option>
-			    		<form:option value="Virtual">Virtual</form:option>
+			    	<form:select path="tipo" class="form-control" onchange="showHideElement(this.value)">
```
Comment (brunomateus) [inline aplicacoes/gal_npi/src/main/webapp/WEB-INF/views/titulo/adicionar.jsp]: O ideal é que não haja código JS misturado no HTML. Dessa forma, o ideal seria pegar elemento via JS, ex: document.getElementById, ou usando o Jquery, e lá anexar o função que irá tratar evento, no caso showHideElement.

File: aplicacoes/gal_npi/src/main/webapp/WEB-INF/views/titulo/editar.jsp [hunk 4]
```diff
@@ -118,14 +119,21 @@
 			<div class="form-group">
 			    <label for="tipo" class="col-sm-1 control-label">Tipo</label>
 			    <div class="col-sm-10">
-			    	<form:select path="tipo" class="form-control">
-			    		<form:option value="Físico">Físico</form:option>
-			    		<form:option value="Virtual">Virtual</form:option>
+			    	<form:select path="tipo" class="form-control" onchange="showHideElement(this.value)">
```
Comment (brunomateus) [inline aplicacoes/gal_npi/src/main/webapp/WEB-INF/views/titulo/editar.jsp]: Mesma situação do comentário anterior.

**Pull request merged**